mesa/README.md

```diff
@@ -1 +1,4 @@
-# MESA
\ No newline at end of file
+# MESA
+
+# FRONT-END DEPENDENCIES:
+# react-native-elements
```

Comment:
You probably want to create a list here, which is done using:
`+` for an unordered list and just numbers `1.` for a ordered list. 

If it is a header you also want to add `##` before it so we keep an structure in the different sections.